pkg/scheduler/server/internal/pool.go

```diff
@@ -182,7 +185,13 @@ func (p *Pool) getConn(meta *schedulerv1pb.JobMetadata) (*conn, error) {
 
 	nsPool, ok := p.nsPool[meta.GetNamespace()]
 	if !ok {
-		return nil, fmt.Errorf("no connections available for namespace: %s", meta.GetNamespace())
+		// no connections available for namespace
+		// TODO: add a dead-letter go routine and log once, but
+		// don't err and spam users if the job can't be sent back
+		if issueSendingJobToDapr.CompareAndSwap(false, true) {
+			log.Warn("No daprd connections available to send triggered job back to")
```

Comment:
This should always be logged at the debug level at least, and should probably include information as to why.

I don't see the need to log this once as a warning as it's entirely client/job specific.